review/tests/test_review_update.py

```diff
@@ -0,0 +1,11 @@
+# import pytest
+# from django.urls import reverse
+#
+#
+# @pytest.mark.django_db
+# class TestUpdate:
+#     def test_review_update(self, client, professional):
+#         url = reverse('review-update', kwargs={'pk': professional.pk})
+#         response = client.get(url)  # This breaks the code
+#         assert response.status_code == 200
+#         assert 'review/review_form.html' in response.templates[0].name
```

Comment:
Why do you need this file if everything is here in the comments?

Reply:
I will fix it